bundle/manifests/observability-operator.clusterserviceversion.yaml

```diff
@@ -59,40 +59,68 @@ spec:
   apiservicedefinitions: {}
   customresourcedefinitions:
     owned:
-    - kind: AlertmanagerConfig
+    - description: AlertmanagerConfig defines a namespaced AlertmanagerConfig to be
```

Comment:
Did you take a look at the UI rendering? I think it should be shorter to fit into the box, like in Gabriel's example

Reply:
It is ok, thanks!

Reply:
<img width="1076" alt="image" src="https://github.com/rhobs/observability-operator/assets/3005132/a2aad4b0-2d3c-4db3-9d13-91fb838b9a1e">

Reply:
You can use  https://operatorhub.io/preview to preview the bundle/csv.yaml 
I think we can improve the  `AlertmanagerConfig` a bit ..  (ask chatgpt :D )

Reply:
BTW, are we now supporting all these API?

Reply:
In a cluster this is the result:
![image](https://github.com/rhobs/observability-operator/assets/91725592/67ea059d-5692-4dbd-b1e9-e5cae1f94194)

Reply:
`"AlertmanagerConfig is used to define configurations for Prometheus Alertmanager, specifying how alerts are handled and routed for notifications."`

Something like this?

Reply:
@simonpasquier would be the best judge of this :) .. BTW, for the record, this was only a suggestion and shouldn't block the PR from merging.

Reply:
small suggestion compared to @marioferh 

> ... specifying how alerts are grouped and notified to external systems.

Reply:
@simonpasquier https://github.com/prometheus-operator/prometheus-operator/pull/5955